docs/source/getting-started.rst

```diff
@@ -2,10 +2,13 @@
 Getting Started
 ===============
 
-This section provides an introduction on how to get started with using the SymbiFlow toolchain.
-In order to generate a bitstream (or any intermediate file format),
-you can use one of the toolchain tests. Following steps describe the whole
-process:
+.. note::
+
+   If you are looking for user documentation, please look at https://github.com/SymbiFlow/symbiflow-examples instead.
```

Comment:
I think "user" documentation means different things to different people. A user could be someone who just uses the FPGAs rather than a person who develops FPGA RTL.

Maybe expand on "If you wish to develop code for an FPGA..."